fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/StandardIndexMaintainer.java

```diff
@@ -359,6 +363,26 @@ public void updateUniquenessViolations(@Nonnull Tuple valueKey, @Nonnull Tuple p
         }
     }
 
+    @Nonnull
+    @Override
+    public CompletableFuture<Void> removeUniquenessViolationsAsync(@Nonnull Tuple valueKey, @Nonnull Tuple primaryKey) {
+        Subspace uniqueValueSubspace = state.store.indexUniquenessViolationsSubspace(state.index).subspace(valueKey);
```

Comment:
Not for this PR, but I think it would make some sense to cache subspace computations that happen at high frequently (e.g. on every record that is accessed or changed), so we don't compute `state.store.indexUniquenessViolationsSubspace(state.index)` on every key.